tidb-cloud/changefeed-sink-to-apache-kafka.md

```diff
@@ -18,6 +18,12 @@ This document describes how to create a changefeed to stream data from TiDB Clou
 - Currently, TiDB Cloud does not support uploading self-signed TLS certificates to connect to Kafka brokers.
```

Comment:
Preview https://pingcap-docsite-preview-hvfal0j3g-hfxsds-projects.vercel.app/tidbcloud/changefeed-sink-to-apache-kafka